s3/pkg/conf/tidb.sql

```diff
@@ -239,3 +239,8 @@ CREATE TABLE IF NOT EXISTS `lifecycle` (
                        `status` varchar(255) DEFAULT NULL
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_bin;
 
+CREATE TABLE IF NOT EXISTS `bucket_versionopts` (
```

Comment:
Hi It seems, only one status saved for versioning, so why we created a new table bucket_versionopts, but not used bucket table itself?

Reply:
For example, if in future we want to include versionid and version details also, the new table can be of reference easily, hence created new one

Reply:
But... versionid is for object not bucket, what we are talking about is bucket.

Reply:
Buckets can be in one of three states: unversioned (the default), versioning-enabled, or versioning-suspended, and I think    
it is better that we store the version states in table bukets

Reply:
Status is just one of the attributes in bucket version table, in future we may need to add/modify different attributes and each time to alter bucket table is not wise , hence to normalize the bucket table it's better we use different table for versioning query, say for versioning if someone wants to add or modify parameters, they can just modify version table, no changes on listbucket, getbucket modules. just change in versioning related APIs